plugins/commands.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const joi = require('joi');
+const boom = require('boom');
+const requestApi = require('request');
+const SCHEMA_COMMAND_NAMESPACE = joi.string().label('Command namespace');
```

Comment:
Should probably use the data-schema here: https://github.com/screwdriver-cd/data-schema/blob/master/config/command.js#L7